chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -889,20 +883,18 @@ def nonconformity_scores(uncal_preds):
     def calibrate(self):
         uncal_preds = np.array(self.calibration_predictor.get_uncal_preds())  # shape(data, tasks, num_classes)
         targets = np.array(self.calibration_data.targets(), dtype=bool)  # shape(data, tasks)
-        (self.num_data, self.num_tasks) = targets.shape
+        self.num_data, self.num_tasks = targets.shape
 
-        # Calculate calibration scores for in set
         has_zeros = np.sum(targets==0, axis=1) > 0
         inds_zeros = targets[has_zeros] == 0
         scores_in = self.nonconformity_scores(uncal_preds[has_zeros])
-        masked_scores_in = scores_in * inds_zeros + np.nan_to_num(np.Inf * (1 - inds_zeros).astype(float), copy=True, nan=0.0, posinf=None, neginf=None)
+        masked_scores_in = scores_in * inds_zeros + np.nan_to_num(np.inf * (1 - inds_zeros).astype(float), copy=True, nan=0.0, posinf=None, neginf=None)
```

Comment:
```suggestion
        masked_scores_in = scores_in * inds_zeros + np.nan_to_num(np.inf * (1 - inds_zeros).astype(float))
```
all of these arguments are default values